app/assets/javascripts/govuk_publishing_components/components/cookie-banner.js

```diff
@@ -44,7 +40,8 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
   CookieBanner.prototype.showCookieMessage = function () {
     // Hide the cookie banner on the cookie settings page, to avoid circular journeys
-    if (this.$module.cookieBanner && window.location.pathname === '/help/cookies') {
+    // or when presented in an iframe by a publishing application
+    if (this.isInCookiesPage() || (this.isInIframe() && this.parentIsPublishingDomain())) {
       this.$module.style.display = 'none'
```

Comment:
Is this line necessary, if it's hidden by default?

Reply:
I will dig a bit more before removing it, but it seems to not have any effects.

Reply:
Updated refactored the conditional statement and removed it.